### PR TITLE
 Frontend: InteractiveTable fix - prevent styles from leaking into tables added in expanded rows

### PR DESCRIPTION
**Pull request by gtk-grafana**
**What is this feature?**
Fixing bug when trying to include nested tables inside the InteractiveTable, like the logs panel.

**Why do we need this feature?**
Styles for interactive table should not impact dynamic content

**Who is this feature for?**
Developers using the interactive table

**Special notes for your reviewer:**
This shouldn't impact any existing styles, but should prevent unwanted style cascade into custom (table) content within expanded row.

Before:
image
image

After:
image
image

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.tsx [hunk 1]
```diff
@@ -38,12 +38,13 @@ const getStyles = (theme: GrafanaTheme2) => {
...
 
-      td: {
-        padding: theme.spacing(1),
-      },
-
-      'td, th': {
-        minWidth: theme.spacing(3),
+      '> tbody, > thead': {
+        '> tr > td': {
+          padding: theme.spacing(1),
```
Comment (gtk-grafana) [inline packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.tsx:43]: Bug: This now overwrites the padding styles in the expandedContentRow > td. Please fix

Comment (gtk-grafana) [reply packages/grafana-ui/src/components/InteractiveTable/InteractiveTable.tsx]: Removed all generic selectors that weren't already scoped with the child-combinator selector. Expanded row padding is now working as expected.

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍 

thanks for this 🥳

**Comment by gtk-grafana** [comment]
What did I do wrong? 🤦 Why is this not in the v11.1.0 release?

**Pull request merged**